File: templates/Uwp/_comp/_shared/Test.UnitTests.Core.SampleData.ContentGrid.xUnit._VB/Param_ProjectName.Core.Tests.xUnit/Tests_postaction.vb
'{[{
Imports Param_RootNamespace.Core.Services
'}]}

Public Class Tests
    '^^
    '{[{

    ' TODO WTS: Remove or update this once your app is using real data and not the SampleDataService.
    ' This test serves only as a demonstration of testing functionality in the Core library.
    <Fact>
    Public Async Sub EnsureSampleDataServiceReturnsContentGridDataAsync()
        Dim actual = Await SampleDataService.GetContentGridDataAsync()

        Assert.NotEmpty(actual)
    End Sub
    '}]}
End Class

File: templates/Uwp/_comp/Prism/Test.UnitTests.Core.SampleData.ContentGrid.NUnit/Param_ProjectName.Core.Tests.NUnit/Tests_postaction.cs
//{[{
using System.Threading.Tasks;
using Param_RootNamespace.Core.Services;
//}]}

namespace Param_RootNamespace.Core.Tests.NUnit
{
    public class Tests
    {
        //^^
        //{[{

        // TODO WTS: Remove or update this once your app is using real data and not the SampleDataService.
        // This test serves only as a demonstration of testing functionality in the Core library.
        [Test]
        public async Task EnsureSampleDataServiceReturnsContentGridDataAsync()
        {
            var dataService = new SampleDataService();
            var actual = await dataService.GetContentGridDataAsync();

            Assert.AreNotEqual(0, actual.Count());
        }
        //}]}
    }
}
